### ISSUE TITLE
Extra bracket on texture definition

### ISSUE THREAD

**Issue by freds72** [sirjuddington/SLADE#1202]
**SLADE Version**: 3.2.0 beta 2
**Operating System**: Windows

**Issue Details**:
Saving a TEXTURE file (folder archive), the result file contains an extra "}" on each texture entry.

**Repro**
Open texture editor
Change texture settings
Save
Notice extra closing bracket:
```
Flat "WALL", 16, 16
{
	XScale 0.500
	YScale 0.500
	Patch "GRAPHICS/TILES.PNG", -48, 0
}}
```

**Comment by ZedekThePlagueDoctor** [comment]
Same issue, confirmed. 

3.2.0 beta 2, Manjaro Linux.

**Comment by sirjuddington** [comment]
Fixed in cbdfd25
